File: PageantManager.Web/src/app/shared/models/garment-type.ts
import { Garment } from './garment';
import { GarmentMeasurementType } from './garment-measurement-type';
import { MeasurementOption } from './measurement-option';

export class GarmentType {
  garmentTypeId: number;
  name: string;
  description: string;
  photo: string;
  garments: Garment[];
  garmentMeasurementTypes: GarmentMeasurementType[];
  measurementOptions: MeasurementOption[];

  public constructor(init?: Partial<GarmentType>) {
    Object.assign(this, init);
  }
}

File: PageantManager.Core/Entities/GarmentType.cs
using System.Collections.Generic;

namespace PageantManager.Core.Entities
{
    public class GarmentType : Entity
    {
        public int GarmentTypeId { get; set; }
        
        public string Name { get; set; }

        public string Description { get; set; }
        
        public string Photo { get; set; }

        public List<CostumeGarment> CostumeGarments { get; set; }
        public List<Garment> Garments { get; set; }
        public List<GarmentMeasurementType> GarmentMeasurementTypes { get; set; }
    }
}
